services/networking/vpc/features.yaml

```diff
@@ -0,0 +1,38 @@
+common_features:
+  - CCC.F01  # Encryption in Transit Enabled by Default
+  - CCC.F03  # Access/Activity Logs
+  - CCC.F06  # Identity Based Access Control
+  - CCC.F08  # Multi-zone Deployment
+  - CCC.F09  # Monitoring
+  - CCC.F10  # Logging
+
+features:
+  - id: CCC.VPC.F01
+    title: Custom Network Creation
```

Comment:
Should this be renamed as "Isolated Network Creation" because the main feature here is isolation